README.md

```diff
@@ -37,10 +37,23 @@ metadata:
 spec:
   multus: {}
 ```
-
 Additionally, container image used to deliver this plugin can be set using
 `MULTUS_IMAGE` environment variable in operator deployment manifest.
 
+### Multus V3.X
+
+The operator also enable deploying Multus V3.X which is the thin CNI plugin form. It is represented by the `multusV3` attribute.
+```yaml
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1
+kind: NetworkAddonsConfig
+metadata:
+  name: cluster
+spec:
+  multusV3: {}
+```
+
+> **_NOTE:_** Having both `multus` and `multusV3` specified is not supported and should be avoided.
```

Comment:
can it be enforced?

Reply:
I think it's possible, the idea of the current implementation is to keep things simple as possible and avoid the maintenance overhead.

This is an intermediate state until Multus V4 is GA'ed on U/S.